src/views/transaction/transaction.view.jsx

```diff
@@ -41,7 +41,7 @@ function Transaction ({
       case TxType.Withdraw:
       case TxType.Exit:
       case TxType.ForceExit: {
-        return 'Withdraw'
+        return 'Exit (Initiate withdraw - step 1)'
```

Comment:
I have concerns with the label itself. Maybe `Exit (Withdraw Step 1)` The Initiate makes it feel as if the tx hasn't been completed

Reply:
We'll add "Withdraw Step 1/2 Completed" instead.